SnakeAndLadder/src/main/java/Application.java

```diff
@@ -0,0 +1,10 @@
+import services.ForwardPlayingGame;
+import services.Game;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Game game = new ForwardPlayingGame(1, 5, 7, 3);
```

Comment:
Use a strategy here and also a builder patterns

Reply:
Added strategy. All the parameters here are mandatory, builder pattern would only help us avoid managing the order of params. Not using builder for now.